primefaces/src/main/resources/META-INF/resources/primefaces/forms/forms.commandbutton.js

```diff
@@ -3,6 +3,8 @@
  * 
  * CommandButton is an extended version of standard commandButton with AJAX and theming.
  *
+ * @prop {number} [ajaxCount] number of concurrent active Ajax requests.
```

Comment:
I think to start your sentence the "n" in Number" has to be capitilized its failing "lint" check right now